custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/README.md

```diff
@@ -61,6 +60,58 @@ will create a custom-metric-prometheus-sd deployment containing prometheus_dummy
 Additionally it will create the HPA object to tell Kubernetes to scale the deployment based on the exported metric. Default maximum number of
 pods is 5.
 
+## Google Managed Prometheus
```

Comment:
Please use official name "Google Cloud Managed Service for Prometheus" for all references. "Google Managed Prometheus" and "GMP" are less preferred